assets/app/views/settings.html

```diff
@@ -72,24 +72,48 @@
       <h2>
         <span>Resource limits</span>
         <i style="margin-left: 5px; cursor: help;" class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
-          title="Resource limits define minimum and maximum values that can be created for each resource type."></i>
+          title="Resource limits define minimum and maximum values that can be requested for each resource type."></i>
       </h2>
       <div ng-if="(limitRanges | hashSize) == 0"><em>{{emptyMessageLimitRanges}}</em></div>
       <table ng-if="(limitRanges | hashSize) > 0" class="table">
         <thead>
           <th>Resource type</th>
-          <th>Min</th>
-          <th>Max</th>
-          <th>Default</th>
+          <th>
+            Min
+            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
+          title="The minimum amount of this resource that can be requested."></i>
+          </th>
+          <th>
+            Max
+            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
+          title="The maximum amount of this resource that can be requested.  The limit must also be below the maximum."></i>
+          </th>
+          <th>
+            Default Request
+            <i class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
+          title="The request is the amount of the resource that will be applied against your quota, you will be guaranteed to have that much of the resource available.  The default request will be applied to all newly created objects unless another request is specified."></i>
```

Comment:
A container will default to request this amount of a compute resource if no request is specified.  The system will guarantee the requested amount of compute resource when scheduling a container for execution.  If a quota is enabled for this compute resource, the quota usage is incremented by the requested value.